fastapi/utils.py

```diff
@@ -39,10 +39,9 @@ def get_model_definitions(
 ) -> Dict[str, Any]:
     definitions: Dict[str, Dict] = {}
     for model in flat_models:
-        m_schema, m_definitions = model_process_schema(
+        m_schema, _ = model_process_schema(
             model, model_name_map=model_name_map, ref_prefix=REF_PREFIX
         )
-        definitions.update(m_definitions)
```

Comment:
I believe the update should ultimately be redundant given the iteration over `flat_models`; either way this fixes a bug with my server.

I don't fully understand why this is the case yet, but I will dig a little deeper.